cyclus_input/predicting_the_past.xml

```diff
@@ -21,6 +22,7 @@
       <Sink>
         <in_commods>
           <val>casked_spent_uox</val>
+          <val>tails</val>
```

Comment:
`caksed_spent_uox` is HLW,
while `tails` is LLW.
The prototype `Sink` is deployed in timestep 960, which is half accurate,
considering there are already LLW (or tails) repositories around.
I suggest creating a separate `sink` for LLW which is present from the beginning of the simulation.